src/cmd/user.go

```diff
@@ -30,9 +30,11 @@ var createUserCmd = &cobra.Command{
 	Long:  "Create a User and optionally define the role (options `User`|`Admin`).",
 	Example: `
 opslevel create user "john@example.com" "John Doe"
+opslevel create user "jane@example.com" "Jane Doe" Admin --skip-send-invite
 opslevel create user "jane@example.com" "Jane Doe" Admin --skip-welcome-email
+opslevel create user "jane@example.com" "Jane Doe" Admin --skip-send-invite --skip-welcome-email
 `,
-	Args: cobra.MinimumNArgs(2),
+	Args: cobra.RangeArgs(2, 4),
```

Comment:
Shouldn't this be `RangeArgs(2, 3)`. The new flag is not a arg only email, name and role are the arguments - the rest are flags.

Reply:
👍 you are spot on. Adding this in